src/main/java/org/sagebionetworks/web/client/jsinterop/mui/Grid.java

```diff
@@ -0,0 +1,91 @@
+package org.sagebionetworks.web.client.jsinterop.mui;
+
+import org.sagebionetworks.web.client.jsinterop.ReactComponentType;
+import org.sagebionetworks.web.client.jsinterop.react.HasStyle;
+
+public class Grid extends HasStyle<ReactComponentType<GridProps>, GridProps> {
+
+  public Grid() {
+    super(MaterialUI.Unstable_Grid2, GridProps.create(false));
+  }
```

Comment:
Note: new constructor pattern for widgets that extend the new ReactComponentV2 class.

Pass the `componentType` and `props`, and rendering will be handled automatically.